### PR TITLE
Updated accounts rst

### PR DESCRIPTION
**Pull request by nrauso**
Modified password expiration paragraph

### CODE REVIEW

File: administrator-manual/en/accounts.rst [hunk 1]
```diff
@@ -472,15 +472,20 @@ The default policy is :dfn:`strong`.
 Expiration
 ----------
 
-The  :index:`password expiration` is enabled by default to 6 months from the time when the password is set.
+The :index:`password expiration` is **NOT** enabled by default.
+If enabled, password lifetime is set to 6 months from the time when the password is set.
```
Comment (DavidePrincipi) [inline administrator-manual/en/accounts.rst]: If enabled, it is set according to `Password policies > The Maximum Password Age` value

File: administrator-manual/en/accounts.rst [hunk 2]
```diff
@@ -472,15 +472,20 @@ The default policy is :dfn:`strong`.
...
 
 .. note:: The system will refer to the date of the last password change,
-   if password is older than 6 months, the server will send an email to indicate that password has expired.
+   if password is older than 6 months, the server consider that password has expired.
    In this case you need to change the user password.
    For example, if the last password change was made in January and the activation of the deadline in October,
    the system will assume the password changed in January is expired, and notify the user.
 
 
+.. warning:: Due to some technical limitation, **no email notification related to password expiration** will
```
Comment (DavidePrincipi) [inline administrator-manual/en/accounts.rst]: I'd remove "Due to some technical limitation"

File: administrator-manual/en/accounts.rst [hunk 3]
```diff
@@ -472,15 +472,20 @@ The default policy is :dfn:`strong`.
 Expiration
 ----------
 
-The  :index:`password expiration` is enabled by default to 6 months from the time when the password is set.
+The :index:`password expiration` is **NOT** enabled by default.
+If enabled, password lifetime is set to 6 months from the time when the password is set.
 
 .. note:: The system will refer to the date of the last password change,
-   if password is older than 6 months, the server will send an email to indicate that password has expired.
+   if password is older than 6 months, the server consider that password has expired.
```
Comment (DavidePrincipi) [inline administrator-manual/en/accounts.rst]: "6 months"... it depends on the actual settings (see previos comment)

```suggestion
   if password is older than :guilabel:`Maximum Password Age`, the password is expired.
```

File: administrator-manual/en/accounts.rst [hunk 4]
```diff
@@ -472,15 +472,20 @@ The default policy is :dfn:`strong`.
...
    In this case you need to change the user password.
    For example, if the last password change was made in January and the activation of the deadline in October,
    the system will assume the password changed in January is expired, and notify the user.
```
Comment (DavidePrincipi) [inline administrator-manual/en/accounts.rst]: Reword "notify the user"

File: administrator-manual/en/accounts.rst [hunk 5]
```diff
@@ -472,15 +472,20 @@ The default policy is :dfn:`strong`.
...
 ----------
 
-The  :index:`password expiration` is enabled by default to 6 months from the time when the password is set.
+The :index:`password expiration` is **NOT** enabled by default.
+If enabled, password lifetime is set to 6 months from the time when the password is set.
 
 .. note:: The system will refer to the date of the last password change,
-   if password is older than 6 months, the server will send an email to indicate that password has expired.
+   if password is older than 6 months, the server consider that password has expired.
    In this case you need to change the user password.
```
Comment (DavidePrincipi) [inline administrator-manual/en/accounts.rst]: "you need" => "the user needs" (?)

File: administrator-manual/en/accounts.rst (hunk 1 above)
Comment (nrauso) [reply administrator-manual/en/accounts.rst]: Ok for me

File: administrator-manual/en/accounts.rst (hunk 2 above)
Comment (nrauso) [reply administrator-manual/en/accounts.rst]: OK

File: administrator-manual/en/accounts.rst (hunk 3 above)
Comment (nrauso) [reply administrator-manual/en/accounts.rst]: OK

File: administrator-manual/en/accounts.rst (hunk 4 above)
Comment (nrauso) [reply administrator-manual/en/accounts.rst]: OK

File: administrator-manual/en/accounts.rst (hunk 5 above)
Comment (nrauso) [reply administrator-manual/en/accounts.rst]: I'm not sure what that means..

Comment (nrauso) [reply administrator-manual/en/accounts.rst]: Changed rst content

**Pull request merged**